evaluate/report/csv.go

```diff
@@ -26,50 +27,109 @@ type CSVFormatter interface {
 	Rows() (rows [][]string)
 }
 
-// GenerateCSV returns the whole CSV as string.
-func GenerateCSV(formatter CSVFormatter) (csvData string, err error) {
-	var out strings.Builder
-	csv := csv.NewWriter(&out)
+// EvaluationRecord holds a line of the evaluation CSV.
+type EvaluationRecord struct {
+	// ModelID holds the model id.
+	ModelID string
+	// ModelName holds the model name.
+	ModelName string
+	// ModelCost holds the model cost.
+	ModelCost float64
```

Comment:
Since those things never change (during an evaluation)... makes me wonder if we just move them at some point such that they are added when the raw records are processed. And not pulled along the whole time.

Reply:
I guess it makes sense just adding the costs when we process the `evalution.csv`. AFAIK we don't need the costs during the evaluation, so that's something we can do afterwards.  

Can be done in a follow-up

Reply:
Plz note down in the issue and resolve this.